### ISSUE TITLE
FHIRHelpers ToQuantity incorrectly using unit

### ISSUE THREAD

**Issue by brynrhodes** [cqframework/clinical_quality_language#557]
The FHIRHelpers ToQuantity conversion code uses the "unit" element of the Quantity type in FHIR, it should be using the "code" element, and should only work if system is either unspecified or equal to UCUM (http://unitsofmeasure.org)

**Comment by brynrhodes** [comment]
Here's what it really should be doing:

```
define function ToQuantity(quantity FHIR.SimpleQuantity):
  case
    when quantity is null then null
    when quantity.value is null then null
    when quantity.system is null or quantity.system = 'http://unitsofmeasure.org' then
      System.Quantity {
        value: quantity.value.value,
        unit: quantity.code.value
      }
    else
      Message(null, true, 'MMECalculator.ToQuantity.InvalidFHIRQuantity', ErrorLevel, 'Invalid FHIR Quantity code: ' & quantity.code.value)
  end
```

**Comment by brynrhodes** [comment]
Included in release 1.5.1
